### DISCUSSION TITLE
A Gallery of Interesting Jupyter Books

### DISCUSSION THREAD

**Discussion by mbobra** [executablebooks/meta#153] [Show and tell] [answered]
I thought it might be nice to have a gallery of interesting Jupyter Books, similar to the [gallery of interesting Jupyter Notebooks](https://github.com/jupyter/jupyter/wiki/A-gallery-of-interesting-Jupyter-Notebooks). It may help people developing their own books to learn by example, or simply serve as a source of inspiration. Perhaps this already exists and I've missed it!

If not, I'm happy to start one. Please let me know if I should start a new wiki page [here](https://github.com/jupyter/jupyter/wiki), make a PR to the Executeable Books Project documentation, or something else.

**Comment by chrisjsewell** [comment, marked as answer]
Have you been looking at https://github.com/executablebooks/meta/issues/149 lol, because yes that is exactly the plan 😄 
But yeh happy to have any input/help on exactly how to implement it!

**Reply by mbobra** [reply]
Ah sorry -- I figured it existed somewhere.

**Reply by choldgraf** [reply]
I think @chrisjsewell is just saying that you are reading our minds :-)

I opened up this issue to discuss the gallery: https://github.com/executablebooks/meta/issues/154

**Reply by matthewfeickert** [reply]
@chrisjsewell Is there a scope recommendation for submitting a Jupyter Book to the [Gallery of Jupyter Books](https://executablebooks.org/en/latest/gallery.html)? That is, are Jupyter Books that are user guides for domain specific libraries too niche to be of general interest (i.e. [`pyhf`'s user guide/tutorial](https://pyhf.github.io/pyhf-tutorial/))? Or would any Jupyter Book that is sufficiently built out be of interest for the gallery?

**Reply by chrisjsewell** [reply]
> is there a scope recommendation for submitting a Jupyter Book to the Gallery of Jupyter Books

I believe the current policy is "the more, the merrier" 😄

**Reply by matthewfeickert** [reply]
Great! Thanks. :smile:
